ros2pkg/ros2pkg/verb/create.py

```
@@ -0,0 +1,112 @@
+            nargs='+',
+            default='main.cpp',
+            help='name of the empty cpp executable')
+
+    def main(self, *, args):
+        package_name = args.package_name
+        destination_directory = args.destination_directory
+        build_tool = args.build_tool
+        dependencies = args.dependencies
+        maintainer_email = args.maintainer_email
+        maintainer_name = args.maintainer_name
+        create_cpp_exe = True if args.create_cpp_exe else False
+        cpp_exe_name = args.cpp_exe_name
+
+        print("going to create a new package")
+        print('package name:', package_name)
+        print('destination directory', destination_directory)
+        print('build tool:', build_tool)
+        print('maintainer_email:', maintainer_email)
+        print('maintainer_name:', maintainer_name)
+        print('dependencies:', dependencies)
+        print('create_cpp_exe:', create_cpp_exe)
+        print('cpp_exe_name:', cpp_exe_name)
+
+        package_directory = create_folder(package_name, destination_directory)
+        if not package_directory:
+            return
+
+        package_xml_config = {
+            'package_name' : package_name,
+            'build_tool' : build_tool,
+            'maintainer_email' : maintainer_email,
+            'maintainer_name' : maintainer_name,
+            'dependencies' : dependencies,
+            }
+        create_template_file('package.xml.em', package_directory, 'package.xml', package_xml_config)
+
+        cmakelists_config = {
+            'project_name' : package_name,
+            'dependencies' : dependencies,
+            'create_cpp_exe' : create_cpp_exe,
+            'cpp_exe_name' : cpp_exe_name
+            }
+        create_template_file('CMakelists.txt.em', package_directory, 'CMakelists.txt', cmakelists_config)
+
+        if (args.create_cpp_exe):
+            src_folder = create_folder('src', package_directory)
+            if not src_folder:
+                return
```

Should this really fail silently?